sdk/digitaltwins/digital-twins-core/test/utils/recorderUtils.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { isPlaybackMode } from "@azure/test-utils-recorder";
+import { isNode } from "@azure/core-http";
+import * as dotenv from "dotenv";
+
+if (isNode) {
+  dotenv.config();
+}
+
+export function uniqueString(): string {
```

Comment:
maybe use uuid v4 here?

Reply:
I copied this from other SDKs. I tried with UUID generation but the (if I remember well) there was a problem with it in the authentication.